WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -15,7 +15,11 @@
         <% elsif competition.in_progress? %>
           <%= icon('fas', 'hourglass-half', title: t('competitions.index.tooltips.hourglass.in_progress'), data: { toggle: "tooltip" }) %>
         <% else %>
-          <%= icon('fas', 'hourglass-start', title: t('competitions.index.tooltips.hourglass.starts_in', days: t('common.days', count:(competition.start_date - Date.today).to_i)), data: { toggle: "tooltip" }) %>
+          <% if @by_announcement_selected %>
```

Comment:
I can't make a suggestion over deleted/unmodified lines, but here I think you can "merge" the if in the previous branch and go:
```ruby
elsif @by_announcement_selected
#...
else
#...
end
```